src/main/resources/edu/arizona/sista/reach/biogrammar/events_master.yml

```diff
@@ -106,8 +118,113 @@ rules:
       label: "Methylation"
       verbalTriggerLemma: "methylate"
       nominalTriggerLemma: "methylation"
+      triggerPrefix: "!word=/(?i)^de/"
+      priority: "5"
+
+
+  #
+  # "Reverse" Simple Events:
+
+  # Dephosphorylations
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Dephosphorylation"
+      actionFlow: "mkBioMention"
+      label: "Dephosphorylation"
+      verbalTriggerLemma: "de-?phosphorylate"
+      nominalTriggerLemma: "de-?phosphorylation"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Deubiquitination
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Deubiquitination"
+      actionFlow: "mkUbiquitination"
+      label: "Deubiquitination"
+      verbalTriggerLemma: "de-?ubiquitinate"
+      nominalTriggerLemma: "de-?ubiquitination"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Dehydroxylation
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Dehydroxylation"
+      actionFlow: "mkBioMention"
+      label: "Dehydroxylation"
+      verbalTriggerLemma: "de-?hydroxylate"
+      nominalTriggerLemma: "de-?hydroxylation"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Desumoylation
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Desumoylation"
+      actionFlow: "mkBioMention"
+      label: "Desumoylation"
+      verbalTriggerLemma: "de-?sumoylate"
+      nominalTriggerLemma: "de-?sumoylation"
+      triggerPrefix: "word=/(?i)^de/"
       priority: "5"
 
+  # Deglycosylation
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Deglycosylation"
+      actionFlow: "mkBioMention"
+      label: "Deglycosylation"
+      verbalTriggerLemma: "de-?glycosylate"
+      nominalTriggerLemma: "de-?glycosylation"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
```

Comment:
After chatting offline, I think I understand your proposal but I still don't agree with it for the following reasons:
1) I don't see why they should be identical to the corresponding additive event -- they are used in two different rule sets and the change doesn't save anything at all,
2) we are not encoding the prefix in two places: the lemma trigger and the triggerPrefix are not the same pattern; they are not even both patterns: one is a pattern but the other is a "rule fragment" containing a (slightly) different pattern.
3) the pattern and the fragment might be combined but this would be less efficient because:
   a) the template rules are based on a conjunction of tests and
   b) the lemma pattern is always tested first and
   c) the current lemma pattern (de-?) is more efficient because it narrows/restricts the set of lemmas matched by the first match (in the template rules), eliminating the need for the second match (in the template rules) in any case where the word contains phosphorylation (e.g. oxidative-phosphorylation)
4) The lemma prefixes also document what that rule is looking for....whereas using just "phosphorylation" is confusing, since the rule is specifically *not* looking for phosphorylation.